src/bootstrap/lib.rs

```diff
@@ -65,9 +65,15 @@
 //! also check out the `src/bootstrap/README.md` file for more information.
 
 #![deny(warnings)]
+#![feature(associated_consts)]
+#![feature(core_intrinsics)]
```

Comment:
I'm surprised whenever I see dynamism even considered, it's rarely *necessary*. Also, mentions of #41875 make me wonder if that decision should be reverted just to prevent people from making copies of `Any` without the `'static` bound, which is *impossible* to make safe (like I said in the past, the only technology we know of which lets us achieve that is higher-ranked lifetime parametrism).